projects/contango-v2/index.js

```diff
@@ -105,17 +105,20 @@ async function positionsTvl(
     target: contangoLens,
     calls: parts.map(([id]) => id),
     abi: abis.balances,
+    permitFailure: true
```

Comment:
it can be dangerous here, any idea why it is failing?

Reply:
I should check, Contango-v2 relies on multiple leveraged strategies on lend/borrow protocols. I wonder if this might be related to the recent Radiant hack

Reply:
yeah, please check. it is better to have broken adapter than one that returns incorrect tvl

Reply:
Ill do asap tomorrow morning, I'll change the title to WIP status in the meantime